### ISSUE TITLE
Spotify banner appears

### ISSUE THREAD

**Issue by N3ars** [sqrt-minus-one/SpotifyAdBlocker-Windows#4]
Spotify ads to buy the subscription for 9,99$ is now popping randomly, any way to avoid that? thanks !

**Comment by Lyceris-chan** [comment]
try running my script which automatically applies this and let me know if you still experience issues since I have no ads here

**Comment by Lyceris-chan** [comment]
can confirm this happened here as well even after running my script no ads tho so it doesn't seem like to big of an issue

**Comment by sqrt-minus-one** [comment]
Yeah, I just noticed it too, I started using the debug build for now to have the logs and try to figure out what is the cause when it'll show up next time.
